src/components/cursor.js

```diff
@@ -26,6 +26,9 @@ var CANVAS_EVENTS = {
 
 var CANVAS_HOVER_CLASS = 'a-mouse-cursor-hover';
 
+// check cursor move over this treshold
+var CURSOR_MOVE_TRESHOLD = 50;
```

Comment:
missing a h => threshold

Reply:
sorry for the late reply! will make a change over the weekend! also will fix the  CI errors

Reply:
@vincentfretin after trying to do this over this weekend, I think I still need some guidance.

I added a  check in onCursorUp, to not trigger EVENTS.CLICK  if the distance is greater than the treshold, but i'm still failing the CI as evt can be undefined. is there a better way for me to get x and y of the cursor at any given time?